Fastfile

```diff
@@ -3,11 +3,16 @@ fastlane_version "1.55.0"
 default_platform :ios
 
   before_all do
+    if ENV['SLACK_URL'] == nil
+      ENV['SLACK_URL'] = ENV['TAB_SLACK_WEBHOOK_URL']
+    end
   end
 
+
   lane :test do
     setup()
-    scan
+    skip_slack = ENV['SCAN_SLACK_CHANNEL'] == "" || ENV['SCAN_SLACK_CHANNEL'] == nil
```

Comment:
```ruby
ENV['SCAN_SLACK_CHANNEL'].to_s.empty?
```
the `.to_s` is important, without that you'll get a `NoMethodError: undefined method `empty?' for nil:NilClass`.

You might want to also think about handling strings that are only whitespace e.g. "   " and skip slack in that case too. To do that, you can use the [String#strip](https://ruby-doc.org/core-2.2.0/String.html#method-i-strip) method

```ruby
ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
```